### DISCUSSION TITLE
How to set localized language

### DISCUSSION THREAD

**Discussion by Chasingyourdreams** [DevToys-app/DevToys#1237] [General]
Hello bro ,I have installed DevToys Now,but i do not know how to set localized language.

**Comment by veler** [comment]
Here bro
![image](https://github.com/DevToys-app/DevToys/assets/3747805/a9717ebc-2e73-4187-990b-66a9f65a01e4)

**Reply by Chasingyourdreams** [reply]
Thank you,bro
